src/core/Form/InputToggleButton/InputToggleButton.baseStyles.ts

```diff
@@ -20,4 +21,16 @@ export const baseStyles = (theme: SuomifiTheme) => css`
       }
     }
   }
+
+  &[disabled],
+  &:disabled {
+    cursor: not-allowed;
+    &.fi-input-toggle-button {
+      & .fi-input-toggle-button_icon {
+        & .fi-icon-base-fill {
+          fill: ${theme.colors.depthBase};
+        }
+      }
+    }
+  }
```

Comment:
Not a big deal, but I'll suggest that we move this inside of the so called base class, to `fi-input-toggle-button` that is few lines above.

Reply:
I don't think it works though? `.fi-input-toggle-button` is the class for the button element, not the SVG inside it.

Reply:
Seems to work, when I tested to move it like the following:

```tsx
&.fi-input-toggle-button {
  display: flex;
  height: 100%;
  & .fi-input-toggle-button_icon {
    pointer-events: none;
    width: 10px;
    height: 10px;
    & .fi-icon-base-fill {
      fill: ${theme.colors.blackBase};
    }
  }

  &[disabled],
  &:disabled {
    cursor: not-allowed;
    & .fi-input-toggle-button_icon {
      & .fi-icon-base-fill {
        fill: ${theme.colors.depthBase};
      }
    }
  }
}
```

Reply:
Ah you meant it like that. Makes very much sense, changed it like so!